website/docs/docs/core/connect-data-platform/databricks-setup.md

```diff
@@ -76,14 +76,14 @@ your_profile_name:
   outputs:
     dev:
       type: databricks
-      catalog: [optional catalog name if you are using Unity Catalog]
-      schema: [schema name] # Required
-      host: [yourorg.databrickshost.com] # Required
-      http_path: [/sql/your/http/path] # Required
+      catalog: CATALOG_NAME #optional catalog name if you are using Unity Catalog
+      schema: SCHEMA_NAME # Required
+      host: YOUR_ORG.databrickshost.com # Required
+      http_path: /SQL/YOUR/HTTP/PATH # Required
       auth_type: oauth # Required if using OAuth-based authentication
-      client_id: [OAuth-Client-ID] # The ID of your OAuth application. Required if using OAuth-based authentication
-      client_secret: [XXXXXXXXXXXXXXXXXXXXXXXXXXX] # OAuth client secret. # Required if using OAuth-based authentication
-      threads: [1 or more]  # Optional, default 1
+      client_id: OAUTH_CLIENT_ID # The ID of your OAuth application. Required if using OAuth-based authentication
+      client_secret: XXXXXXXXXXXXXXXXXXXXXXXXXXX # OAuth client secret. # Required if using OAuth-based authentication
+      threads: 1_OR_MORE  # Optional, default 1
 ```
 </File>
```

Comment:
I think we should change the options that show up in the tables below. Like [here](https://docs-getdbt-mdk5plxur-dbt-labs.vercel.app/docs/core/connect-data-platform/databricks-setup?tokenoauth=oauth#host-parameters) entries in the [authentication parameters table](https://docs-getdbt-mdk5plxur-dbt-labs.vercel.app/docs/core/connect-data-platform/databricks-setup?tokenoauth=oauth#authentication-parameters) might be confusing especially now they don't match the example.

Reply:
Good call!